helm_deploy/values-dev.yaml

```diff
@@ -13,6 +13,14 @@ generic-service:
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
+  allowlist:
```

Comment:
Don't you need to add the others from https://github.com/ministryofjustice/offender-case-notes/blob/main/helm_deploy/offender-case-notes/values.yaml#L58-L66